dotcom-rendering/src/components/KeyTakeaways.tsx

```diff
@@ -21,9 +21,11 @@ interface KeyTakeawaysProps {
 	starRating?: number;
 	keyTakeaways: KeyTakeaway[];
 	RenderArticleElement: ArticleElementRenderer;
+	// is this the last element in the article?
```

Comment:
If you use JSDoc here it will be available in IDEs as information about this prop